gpii/node_modules/processReporter/README.md

```diff
@@ -0,0 +1,68 @@
+Process Reporter
+===
+
+A server app built deployed with a node.js server that returns process information.
+
+### Process Reporter API
+
+The app currently supports the following urls:
+
+    {urlToProcessReporterServer}/processes            // GET ONLY
+    {urlToProcessReporterServer}/processes/{solution} // GET ONLY
+
+The first form cycles through the solutions registry, and for each entry, determines if the associated processes are running.  The second form reports only the solution requested, for example:
+
+    {urlToProcessReporterServer}/processes/org.gnome.desktop.a11y.magnifier
+
+In both cases, the Process Reporter uses the information stored in the `isRunning` block of a solutions registry entry.  For example, the `isRunning` block for the Orca screen reader (the solution entry `org.gnome.orca`) is as
```

Comment:
Didn't we decide that a solution without an "isRunning" block would be considered to be always running?